src/index.js

```diff
@@ -33,11 +34,12 @@ class Index extends React.Component {
         <Switch>
           <Route exact path='/login' component={Login} />
           <Route exact path='/register' component={Register} />
-          <Route exact path='/accueil' component={Popular} />
-          <Route exact path='/top_rated' component={TopRated} />
+          <Route exact path='/accueil' component={Popular} match={this.props.match} />
```

Comment:
Not work... Need to do like this
```javascript
component={(match, location, history) => {
  <Popular match={match} />
}}
```

Reply:
Expected an assignment or function call and instead saw an expression. (no-unused-expressions)

Reply:
Mister "I know everything" !!

Reply:
... Plus teubé que toi c'est rare

Reply:
J'te laisse chercher